roles/job_runner/tasks/credentials.yml

```diff
@@ -34,6 +34,16 @@
     - lookup('env','PASSWORD_SECRET') == ''
     - lookup('env','TOKEN_SECRET') != ''
 
+- name: Launch Container Registry Credential
+  include_tasks: credentials/create_token_credential.yml
+  when:
+    - lookup('env','KUBERNETES_BEARER_TOKEN') == ''
+    - lookup('env','SSH_SECRET') == ''
+    - lookup('env','USERNAME_SECRET') != ''
+    - lookup('env','PASSWORD_SECRET') != ''
+    - lookup('env','CONTAINER_REGISTRY_HOST') != ''
+    - lookup('env','TOKEN_SECRET') == ''
+  
```

Comment:
@the-it-jaeger It looks like CI is just complaining about this extra whitespace here, can you remove this line?

Reply:
Sure. That linter is picky, eh?